### ISSUE TITLE
Cookie import error

### ISSUE THREAD

**Issue by v-moravec** [Hebilicious/authjs-nuxt#2]
Hi,

I'm getting this error with this module:

`Uncaught SyntaxError: The requested module '/_nuxt/node_modules/cookie/index.js?v=41251351' does not provide an export named 'parse' (at web.js?v=41251351:1:10)`

When I tried to implement it myself, I don't get this error (it works locally), but I can't build for CF Pages, because of `crypto` dependency, which I can't get rid of.

**Comment by Hebilicious** [comment]
@v-moravec Hi, could you provide me with a repo or a stackblitz with your issues ? I have successfully been able to publish a website on CF pages with this module.

**Comment by v-moravec** [comment]
Yes that solved it, thanks. I suspect, it might have something to do with the `@auth/core` version, since it doesn't happen to me with '0.8.1'. I also solved the problem with my setup - I had to add this:

```
nitro: {
  rollupConfig: {
    external: [ '@auth/core']
  }
}
```

And now it works on edge runtime.

**Comment by Hebilicious** [comment]
@v-moravec This should be fixed with https://github.com/Hebilicious/authjs-nuxt/commit/6dde4c3d4acac044f44ac7e6d1f66317aff47db5 
Please let me know if you're still having issues with the latest versions.

**Comment by Hebilicious** [comment]
I think it has to do with something wrong within @auth/core and therefore I added the workaround to the documentation. I'll leave this open just in case there's a better solution down the line.

**Comment by dissy123** [comment]
Hi,

I installed the cookie package also via pnpm i and then added an alias, and it is finding the package. 

But now i get: 

` [auth][error][InvalidCheck]: Read more at https://errors.authjs.dev#invalidcheck
   [auth][cause]: InvalidCheck: PKCE code_verifier cookie was missing.
`

is this related to that bug? 
on the login page is only one cookie set that one with the redirect url.

Would be great if someone has an idea on this :)

**Comment by Hebilicious** [comment]
> Hi,
> 
> I installed the cookie package also via pnpm i and then added an alias, and it is finding the package.
> 
> But now i get:
> 
> `[auth][error][InvalidCheck]: Read more at https://errors.authjs.dev#invalidcheck [auth][cause]: InvalidCheck: PKCE code_verifier cookie was missing.`
> 
> is this related to that bug? on the login page is only one cookie set that one with the redirect url.
> 
> Would be great if someone has an idea on this :)

Can you please open a new issue with a reproduction ?

**Comment by Q16solver** [comment]
Should this be raised as an issue in the authjs repo?

**Comment by Hebilicious** [comment]
> @Hebilicious Should this be raised as an upstream issue in the authjs repo?

I think it could yes.

**Comment by ben05allen** [comment]
I have what seems to be a related issue:
`Uncaught SyntaxError: ambiguous indirect export: parse [web.js:1:10](http://localhost:3000/_nuxt/node_modules/@auth/core/lib/web.js?v=4475a7ed)`

`
- Operating System: Linux
- Node Version:     v18.18.1
- Nuxt Version:     3.9.3
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@9.8.1
- Builder:          -
- User Config:      devtools, modules, authJs, nitro, experimental, runtimeConfig
- Runtime Modules:  @nuxtjs/tailwindcss@6.11.1, @hebilicious/authjs-nuxt@0.3.5
- Build Modules:    -
`

**Comment by niko-chaffinchicas** [comment]
> I am not sure why this bug is happening, but as a workaround, you can try to install cookie, and to add this to your package.json :
> 
> ```ts
> export default defineNuxtConfig({
>   alias: {
>     cookie: path.resolve(__dirname, "node_modules/cookie")
>   }
> })
> ```

@Hebilicious Thank you for this, this had been causing issues for me as well.

It didn't look like anyone has raised an issue upstream with `@auth/core` yet, and the latest version `0.30.0` still has this issue. Shall I go ahead and open that issue?
